.gitignore

```diff
@@ -18,7 +18,7 @@
 # misc
 .DS_Store
 *.pem
-yarn.lock
+package-lock.json
```

Comment:
I'm not sure if it's a hard and fast rule to not edit package-lock.json though. Need to investigate this. Generally I do try to leave it out.

Reply:
I think repos that have their own npm packages should be executed with `npm` commands (i.e. v1-core, v1-periphery). Repos that don't have their own packages (i.e. v1-interface, v1-strategies, etc.), should be executed with `yarn`. We are trying to optimize download speeds and reduce dependency issues so yarn helps a lot with that. For consistency, we should hold npm packages to be used with their native commands.